code/obj/machinery/computer/QM_order.dm

```diff
@@ -107,26 +109,39 @@
 																						1px -1px 0 #000,
 																						-1px 1px 0 #000,
 																						 1px 1px 0 #000;}
-		</style>"}
+		</style>
+		<script type="text/javascript">
+			// apparently just normal ol "a href=#fuck" links dont work in byond
+			// im at a loss for words
+			function are_you_fucking_shitting_me(h) {
```

Comment:
Actually meaningful function names, please. Pre-responding to "this is a meaningful function name": yes, very good. Now actually change it. :)

Reply:
should i change this in QM_supply as well?

Reply:
Sure.